engine/src/main/java/org/hibernate/search/engine/backend/analysis/AnalyzerNames.java

```diff
@@ -19,14 +19,59 @@ private AnalyzerNames() {
 	 * <p>
 	 * This analyzer will generally be used for full-text field that don't require specific analysis.
 	 * <p>
-	 * Unless overridden by explicit analysis configuration, the default analyzer will be the standard analyzer:
+	 * Unless overridden by explicit analysis configuration,
+	 * the default analyzer will be the {@link #STANDARD} analyzer.
```

Comment:
That's not really what you implemented, though. If a user overrides the analyzer named "standard", the analyzer named "default" will still be the `StandardAnalyzer`, not the user defined analyzer for "standard".

Maybe keep the previous documentation for the default analyzer, to be more correct?

Reply:
You're right, if the user overrode the `standard` analyzer, this sentence would no longer be true!
I'm restoring the old documentation...